### PR TITLE
This action scans your pull requests for dependency changes

### PR DESCRIPTION
**Pull request by elivan-k**
## What

This action scans your pull requests for dependency changes, and will raise an error if any vulnerabilities or invalid licenses are being introduced.

## Why

This is a part of the GitHub Advanced Security to improve our Security in Github.

## References

Related to Jira [DEVOPS-624](https://jira.greenbone.net/browse/DEVOPS-624)
More info [dependency-review-action](https://github.com/actions/dependency-review-action)

### CODE REVIEW

File: dependency-review-action/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  contents: read
+
+jobs:
+  dependency-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout Repository'
+        uses: greenbone/actions/dependency-review-action@v1
+      - name: 'Dependency Review'
+        uses: greenbone/actions/dependency-review-action@v1
```
Comment (bjoernricks) [inline dependency-review-action/README.md:27]:
```suggestion
      - name: 'Dependency Review'
        uses: greenbone/actions/dependency-review-action@v2
```

Comment (elivan-k) [reply dependency-review-action/README.md:27]: Also change in line 27 to v2 right?

Comment (bjoernricks) [reply dependency-review-action/README.md:27]: Yes exactly

File: dependency-review/README.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+permissions:
+  contents: read
+
+jobs:
+  dependency-review:
+    runs-on: ubuntu-latest
+    steps:
+      - name: 'Checkout Repository'
+        uses: greenbone/actions/dependency-review@v2
```
Comment (bjoernricks) [inline dependency-review/README.md:25]:
```suggestion
```

**Review by bjoernricks — APPROVED** [review]

**Pull request merged**